docs/README.md

```diff
@@ -0,0 +1,319 @@
+# Gridlook 2025-07-22
+
+**ESM data viewer on a 3D sphere using WebGL**
+
+Andrej Fast¹, Tobi Kölling², Fabian Wachsmann¹, Lukas Kluft²
+
+¹DKRZ, ²MPI-M
+
+🔗 [Github](https://github.com/d70-t/gridlook) - 🔗 [Deployed](https://gridlook.pages.dev/)
+
+---
+
+## 🌍 Motivation
+
+Build an easy-to-use **visualisation tool** to:
+
+- 👀 show plotting without HPC
+- 🔎 *Simply and Interactively explore*
+    **native grid** Earth System Model (ESM) output
+- 🔗 Share *any dataset view* via URL
+- 🚅💨 no installation or compute server required
+
+----
+
+**🌍 Motivation**
+
+for developers: interesting technology!
+
+🎨 *client-side* rendering and color mapping
+**no image pregeneration**
+
+
+----
+
+## ☁️ Set-up
+
+Support and leverage **any Zarr dataset** stored in cloud environments.
+
+📌 **Recipe**:  
+`https://gridlook.pages.dev/#` + `ZARR_URI`
+
+Where `ZARR_URI`:
+
+- ✅ Is **openly accessible**
+- 🌐 Allows **Cross-Origin Requests (CORS)**
+
+
+----
+
+## Gridlook is *Not*
+
+- ❌ A competitor to high-end visualization suites
+- ❌ A tool for generating publication-quality graphics<br>
+👉 It’s built for intuitive, efficient, interoperable **exploration**.
+
+---
+
+## 🛠 Technical Details
+
+- 🧠 Built with **TypeScript**
+- 🌱 Frontend: **Vue.js** + **Bulma**
+- 🔺 Rendering: **Three.js (WebGL)**
+- 📦 Zarr handling via **Zarrita**
+
+----
+
+## Supported Grids
+
+- 🌐 Regular Grids
+- 🧭 Rotated Regular Grids
+- 💠 HEALPix
+- 🔺 Triangular (ICON)
+- 🪢 * Irregular Grids
+- 🌐 * Gaussian Reduced (e.g., ERA5)
+<br/>
+<div style="font-size:10pt;">
+* Not upstreamed yet
+</div>
+
+----
+
```

Comment:
remove limitations. We have the issue tracker for that

Reply:
I linked it to issues. Is that OK?

Reply:
I'd just remove the section.